project/src/pages/not-found/not-found.tsx

```diff
@@ -0,0 +1,34 @@
+import { useNavigate } from 'react-router-dom';
+import styles from './not-found.module.css';
+import Header from '../../components/header/header';
+
+function NotFound(): JSX.Element {
+  const navigate = useNavigate();
+
+  return (
+    <div className="page page--gray">
+      <Header isLogged={false} />
+
+      <main className={`${styles.page__error} error`}>
```

Comment:
поправив классы в цсс ты сможешь писать
```suggestion
      <main className={styles.pageError`}>
```